packages/braintree-utils/src/utils/index.ts

```diff
@@ -1,3 +1,6 @@
 export { default as getValidBraintreeConnectStyles } from './get-valid-braintree-connect-styles';
 export { default as isBraintreeConnectWindow } from './is-braintree-connect-window';
 export { default as isBraintreeAcceleratedCheckoutCustomer } from './is-braintree-accelerated-checkout-customer';
+
+// Fastlane
+export { default as getValidBraintreeFastlaneStyles } from './get-valid-braintree-fastlane-styles';
```

Comment:
It is not necessary to create new method for styling, it might be better to rename getValidBraintreeConnectStyles and update props interfaces

Reply:
renamed